src/peepsController.js

```diff
@@ -0,0 +1,214 @@
+$(document).ready(function() { 
+    //loads all peeps when the page is loaded
+  fetch('https://chitter-backend-api.herokuapp.com/peeps')
+    .then(function(response) {
```

Comment:
Nice! This could be encapsulated into one callback function, which could be reused when you call the post method